src/Lean/Server/FileWorker/RequestHandling.lean

```diff
@@ -38,15 +38,13 @@ def handleCompletion (p : CompletionParams)
   -- NOTE: use `+ 1` since we sometimes want to consider invalid input technically after the command,
   -- such as a trailing dot after an option name. This shouldn't be a problem since any subsequent
   -- command starts with a keyword that (currently?) does not participate in completion.
```

Comment:
This doesn't seem to be true anymore as the subsequent syntax can now also be a tactic keyword and they do participate in completion

Reply:
I changed the implementation to use the syntax endpos now, which (I think) should not cause any issues.